recipe/build.sh

```diff
@@ -1,9 +1,14 @@
 set -ex
 
+# libdir is set to lib, otherwise on aarch64 and some others 
+# will default to lib64
+
 meson setup builddir          \
     ${MESON_ARGS}             \
     -Denable_tests=false      \
-    --buildtype=release
+    --prefix=$PREFIX          \
```

Comment:
isn't --prefix = $PREFIX by default ?

Reply:
We had a bug where it wasn't being applied correctly, and it's better to be explicit than implicit! We really need to stop relying on implicit stuff.

Reply:
Having a look at the logs for both `osx-arm64` and `linux-aarch64`, I can see that for `osx-arm64` the variable `$MESON_ARGS` is set with both `--prefix` and `-Dlibdir` (and the build succeeds), while for `linux-aarch64` the variable `$MESON_ARGS` is not set.

`osx-arm64`:
```
INFO: activate_clang_osx-arm64.sh made the following environmental changes:
...
+MESON_ARGS=--buildtype release --prefix=$PREFIX -Dlibdir=lib
...
```

`linux-aarch64`:
```
INFO: activate-gcc_linux-aarch64.sh made the following environmental changes:
...
<stuff>
...
```

Maybe I should investigate why this is happening.

Reply:
Explicit is better than implicit, so don't waste time investigating this.

Reply:
Cool!